process.go

```diff
@@ -27,6 +27,8 @@ import (
 	"github.com/containerd/containerd/cio"
 	"github.com/containerd/containerd/errdefs"
 	"github.com/containerd/containerd/protobuf"
+	"github.com/containerd/containerd/tracing"
+	"go.opentelemetry.io/otel/attribute"
```

Comment:
Let's not import opentelemetry everywhere (see my comment above).
We should probably use `tracing.Attribute` instead with proper types passed in.